rabbit_monitor.py

```diff
@@ -20,76 +19,121 @@
 
 Settings = namedtuple('Settings',
                       [
+                          'port',
                           'rabbit_url',
                           'rabbit_default_user',
                           'rabbit_default_pass',
+                          'rabbit_default_vhost',
                           'wait_time',
                       ])
 
+logger.info("Creating RabbitMonitor object")
+
+settings = Settings(port=os.getenv("PORT", 5000),
+                    wait_time=settings.WAIT_TIME,
+                    rabbit_url=settings.RABBIT_URL,
+                    rabbit_default_user=settings.RABBITMQ_DEFAULT_USER,
+                    rabbit_default_pass=settings.RABBITMQ_DEFAULT_PASS,
+                    rabbit_default_vhost=settings.RABBITMQ_DEFAULT_VHOST)
+
+healthcheck_url = settings.rabbit_url + 'healthchecks/node'
+aliveness_url = (settings.rabbit_url +
+                 'aliveness-test/{}'.format(settings.rabbit_default_vhost))
+
+urls = {'healthcheck': healthcheck_url,
+        'aliveness': aliveness_url}
+
+"""
+def shutdown():
+    logger.info("Shutting down sdx-rabbit-monitor")
+    sys.exit()
+
+"""
+
+
```

Comment:
Maybe group the coroutines under a Monitor class so it's clear which are related.